contract/src/platform-goals/start-governed-contract.js

```diff
@@ -0,0 +1,280 @@
+  ]);
+  /** @type {GovernanceFacetKit<SF>} */
+  const facets = harden({
+    instance,
+    publicFacet,
+    governor: governorFacets.instance,
+    creatorFacet,
+    adminFacet,
+    governorCreatorFacet: governorFacets.creatorFacet,
+    governorAdminFacet: governorFacets.adminFacet,
+  });
+  return facets;
+};
+
+/**
+ * @param {*} creatorFacet
+ * @param {ERef<NameHub>} namesByAddress
+ * @param {Record<string, string>} voterAddresses
+ */
+export const inviteToMyCharter = async (
+  creatorFacet,
+  namesByAddress,
+  voterAddresses,
+) =>
+  Promise.all(
+    values(voterAddresses).map(async addr => {
+      const debugName = `econ charter member ${addr}`;
+      const depositFacet = E(namesByAddress).lookup(addr, 'depositFacet');
+      const invitation = await E(creatorFacet).makeCharterMemberInvitation();
+      return E(depositFacet)
+        .receive(invitation)
+        .catch(err => console.error(`failed deposit to ${debugName}`, err));
+    }),
+  );
+harden(inviteToMyCharter);
+
+/**
+ * @param {string} contractName
+ * @param {BootstrapPowers} powers
+ * @param {*} config
+ */
+export const startMyCharter = async (contractName, powers, config) => {
+  const committeeName = `${contractName}Committee`;
+  const charterName = `${contractName}Charter`;
+
+  const {
+    consume: { namesByAddress, zoe },
+    produce: { [`${charterName}Kit`]: produceKit },
+    installation: {
+      consume: { binaryVoteCounter: counterP, econCommitteeCharter: installP },
+    },
+    instance: {
+      produce: { [charterName]: instanceP },
+    },
+  } = powers;
+  const {
+    [committeeName]: { voterAddresses },
+  } = config?.options || {};
+  assert(voterAddresses, 'no voter addresses???');
+
+  const [charterInstall, counterInstall] = await Promise.all([
+    installP,
+    counterP,
+  ]);
+  const terms = await allValues({
+    binaryVoteCounterInstallation: counterInstall,
+  });
+
+  const startResult = await E(zoe).startInstance(
+    charterInstall,
+    undefined,
+    terms,
+    undefined,
+    'econCommitteeCharter',
+  );
+  instanceP.resolve(startResult.instance);
+  produceKit.resolve(startResult);
+
+  await inviteToMyCharter(
+    startResult.creatorFacet,
+    namesByAddress,
+    voterAddresses,
+  );
+  return startResult;
+};
+harden(startMyCharter);
+
+/**
+ * @param {string} contractName
+ * @param {BootstrapPowers} powers
+ * @param {*} config
+ */
+export const startMyCommittee = async (contractName, powers, config) => {
+  const committeeName = `${contractName}Committee`;
+  const {
+    consume: { board, chainStorage, namesByAddress, startUpgradable },
+    produce: { [`${committeeName}Kit`]: produceKit },
+    installation: {
+      consume: { committee },
+    },
+    instance: {
+      produce: { [committeeName]: produceInstance },
+    },
+  } = powers;
+  const {
+    [committeeName]: { voterAddresses },
+  } = config?.options || {};
+  assert(voterAddresses, 'no voter addresses???');
```

Comment:
This assert always passes as long as `voteAddresses` is an empty object `{}`. Should we assert that it contains some values?

Reply:
Well, yes, we should... but I wouldn't be surprised if some tests would need work to meet that constraint.